Sources/KeyboardShortcuts/Shortcut.swift

```diff
@@ -179,113 +179,487 @@ extension KeyboardShortcuts.Shortcut {
+	fileprivate var presentableDescription: String {
+		switch self {
+		case .return:
+			"↩"
+		case .delete:
+			"⌫"
+		case .deleteForward:
+			"⌦"
+		case .end:
+			"↘"
+		case .escape:
+			"⎋"
+		case .help:
+			"?⃝"
+		case .home:
+			"↖"
+		case .space:
+			"space_key".localized.capitalized // This matches what macOS uses.
+		case .tab:
+			"⇥"
+		case .pageUp:
+			"⇞"
+		case .pageDown:
+			"⇟"
+		case .upArrow:
+			"↑"
+		case .rightArrow:
+			"→"
+		case .downArrow:
+			"↓"
+		case .leftArrow:
+			"←"
+		case .f1:
+			"F1"
+		case .f2:
+			"F2"
+		case .f3:
+			"F3"
+		case .f4:
+			"F4"
+		case .f5:
+			"F5"
+		case .f6:
+			"F6"
+		case .f7:
+			"F7"
+		case .f8:
+			"F8"
+		case .f9:
+			"F9"
+		case .f10:
+			"F10"
+		case .f11:
+			"F11"
+		case .f12:
+			"F12"
+		case .f13:
+			"F13"
+		case .f14:
+			"F14"
+		case .f15:
+			"F15"
+		case .f16:
+			"F16"
+		case .f17:
+			"F17"
+		case .f18:
+			"F18"
+		case .f19:
+			"F19"
+		case .f20:
+			"F20"
+
+		// Representations for numeric keypad keys with   ⃣  Unicode U+20e3 'COMBINING ENCLOSING KEYCAP'
+		case .keypad0:
+			"0\u{20e3}"
+		case .keypad1:
+			"1\u{20e3}"
+		case .keypad2:
+			"2\u{20e3}"
+		case .keypad3:
+			"3\u{20e3}"
+		case .keypad4:
+			"4\u{20e3}"
+		case .keypad5:
+			"5\u{20e3}"
+		case .keypad6:
+			"6\u{20e3}"
+		case .keypad7:
+			"7\u{20e3}"
+		case .keypad8:
+			"8\u{20e3}"
+		case .keypad9:
+			"9\u{20e3}"
+		// There's "⌧“ 'X In A Rectangle Box' (U+2327), "☒" 'Ballot Box with X' (U+2612), "×" 'Multiplication Sign' (U+00d7), "⨯" 'Vector or Cross Product' (U+2a2f), or a plain small x. All combined symbols appear bigger.
+		case .keypadClear:
+			"☒\u{20e3}" // The combined symbol appears bigger than the other combined 'keycaps'
+		// TODO: Respect locale decimal separator ("." or ",")
+		case .keypadDecimal:
+			".\u{20e3}"
+		case .keypadDivide:
+			"/\u{20e3}"
+		// "⏎" 'Return Symbol' (U+23CE) but "↩" 'Leftwards Arrow with Hook' (U+00d7) seems to be more common on macOS.
+		case .keypadEnter:
+			"↩\u{20e3}" // The combined symbol appears bigger than the other combined 'keycaps'
+		case .keypadEquals:
+			"=\u{20e3}"
+		case .keypadMinus:
+			"-\u{20e3}"
+		case .keypadMultiply:
+			"*\u{20e3}"
+		case .keypadPlus:
+			"+\u{20e3}"
 		}
+	}
 
+	@available(macOS 11.0, *)
+	fileprivate var swiftUIKeyEquivalent: SwiftUI.KeyEquivalent? {
+		switch self {
+		case .return:
+			.return
+		case .delete:
+			.delete
+		case .deleteForward:
+			.deleteForward
+		case .end:
+			.end
+		case .escape:
+			.escape
+		case .help:
+			UnicodeScalar(NSHelpFunctionKey).map { KeyEquivalent(Character($0)) }
+		case .home:
+			.home
+		case .space:
+			.space
+		case .tab:
+			.tab
+		case .pageUp:
+			.pageUp
+		case .pageDown:
+			.pageDown
+		case .upArrow:
+			.upArrow
+		case .rightArrow:
+			.rightArrow
+		case .downArrow:
+			.downArrow
+		case .leftArrow:
+			.leftArrow
+		case .f1:
+			UnicodeScalar(NSF1FunctionKey).map { KeyEquivalent(Character($0)) }
```

Comment:
I would make a tiny helper function for this, to avoid all the repetition. Can be used for `case .help:` too.